sample/src/androidTest/java/com/schibsted/spain/barista/sample/AssertionsTest.java

```diff
@@ -84,6 +94,15 @@ public void checkInvisibleViews() {
     assertNotDisplayed("I'm invisible!");
   }
 
+  @Test
+  public void checkInvisibleViewsWithContains() {
```

Comment:
The test naming sounds confusing here too.
Judging by the code, I think it should be named something like `checkInvisibleViewDoesNotContainText` or similar.

Reply:
Done